tiledb/cloud/array.py

```diff
@@ -563,3 +602,150 @@ def apply(
         result_format_version=result_format_version,
         **kwargs,
     ).get()
+
+
+def exec_multi_array_udf_async(
+    func=None,
+    array_list=None,
+    name=None,
+    layout=None,
+    image_name=None,
+    http_compressor="deflate",
+    include_source_lines=True,
+    task_name=None,
+    result_format=rest_api.models.UDFResultType.NATIVE,
+    result_format_version=None,
+    **kwargs
+):
+    """
+    Apply a user defined function to multiple arrays
+    :param func: user function to run
+    :param array_details: dict from array name (tiledb format) to tuple (ranges, attributes)
```

Comment:
Adjust the description to match the new class

Reply:
Adjusted